src/main/java/io/knowledgeassets/myskills/server/security/SecurityService.java

```diff
@@ -0,0 +1,31 @@
+package io.knowledgeassets.myskills.server.security;
+
+import io.knowledgeassets.myskills.server.community.query.CommunityQueryService;
+import org.springframework.security.oauth2.jwt.Jwt;
+import org.springframework.stereotype.Service;
+
+import static io.knowledgeassets.myskills.server.security.JwtClaims.MYSKILLS_USER_ID;
+import static java.util.Objects.requireNonNull;
+
+/**
+ * Service to accumulate security logic used by {@link MySkillsSecurityExpressionRoot}
+ * to make it available withing the whole application.
+ */
+@Service
+public class SecurityService {
+
+	private final CommunityQueryService communityQueryService;
+
+	public SecurityService(CommunityQueryService communityQueryService) {
+		this.communityQueryService = requireNonNull(communityQueryService);
+	}
+
+	public boolean hasCommunityManagerRole(Object principal, String communityId) {
```

Comment:
Could you please add a unit test for this service?

Reply:
Thanks, done.